extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1577,7 +1577,11 @@ export class AmpStoryPage extends AMP.BaseElement {
     const attachmentEl = this.element.querySelector(
       'amp-story-page-attachment, amp-story-page-outlink, amp-story-shopping-attachment'
     );
-    if (!attachmentEl) {
+
+    if (
+      !attachmentEl ||
+      this.element.getElementsByTagName('amp-story-shopping-tag').length === 0
+    ) {
```

Comment:
This means that the open attachment UI will not render if `(no attachment element exists) OR (no shopping tags exist)`. Rather, you'll want the condition to be something more like `(no attachment element exists) OR (a shopping attachment exists AND no shopping tags exist)`

Reply:
Sounds good to me, I added in the additional logic statement.